src/lib/logic/stores/config.ts

```diff
@@ -1,35 +1,33 @@
 import { setContext } from "svelte";
 import { readable } from "svelte/store";
-
-import type { StoreStyles } from "../typing/stores/form.js";
-import type { FormStyles } from "../typing/globals/proptypes.js";
 import type { Config } from "../typing/stores/config.js";
+import { injectGlobal } from "@emotion/css";
```

Comment:
🚫 **[eslint]** <[import/order](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/order.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`@emotion/css` import should occur before type import of `../typing/stores/config.js`
```suggestion
import { injectGlobal } from "@emotion/css";
import type { Config } from "../typing/stores/config.js";
```